src/bundle/Resources/translations/ibexa_user_settings.en.xliff

```diff
@@ -76,16 +76,16 @@
         <target state="new">Browsing</target>
         <note>key: settings.group.browsing.name</note>
       </trans-unit>
-      <trans-unit id="c9a8a7a02d6e5a8526e883fc7a6e3e35f40cf809" resname="settings.group.edit_content.description">
-        <source>settings.group.edit_content.description</source>
-        <target state="new"></target>
-        <note>key: settings.group.edit_content.description</note>
-      </trans-unit>
       <trans-unit id="962f753514ae358c3a7c093f354f213ec553e9b3" resname="settings.group.content_authoring.name">
         <source>Content authoring</source>
         <target state="new">Content authoring</target>
         <note>key: settings.group.content_authoring.name</note>
       </trans-unit>
+      <trans-unit id="c9a8a7a02d6e5a8526e883fc7a6e3e35f40cf809" resname="settings.group.edit_content.description">
+        <source>settings.group.edit_content.description</source>
+        <target state="new"></target>
```

Comment:
This is suspicious but master of translation accepts so approve ;)

Reply:
It is a common situation that the description is empty in user settings

Reply:
Most suspicious for me in here is that it changed places (but yeah, I also wondered if it's ok that it's empty)